das/coordinator_test.go

```diff
@@ -125,13 +134,14 @@ func TestCoordinator(t *testing.T) {
 	})
 
 	t.Run("priority routine should not lock other workers", func(t *testing.T) {
+		params.samplingRange = 20
```

Comment:
@walldiss tests wouldn't pass until I made this. Can you scrutinize these changes?

Reply:
This line will make test pointless. The point is to have priority routine discover new headers and sample recent headers sooner than catchup workers reach it. I'll point to root of the issue in different comment